modules/accessing-microshift-cluster-remotely.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+:_content-type: CONCEPT
+[id="accessing-microshift-cluster-remotely_{context}"]
+= Accessing the MicroShift cluster remotely
+
+Use the following procedure to access the MicroShift cluster remotely. 
```

Comment:
This section is a little confusing because it's not clear what the "remote" host is. Some of the text seems to be written from the perspective of a user working directly on the microshift host, then accessing another host. But some of the matching instructions are suggesting commands from the opposite perspective.

What's the goal for this section?

Reply:
The previous section described how to use `oc` on the system running MicroShift.

This section is about running `oc` from a different system, which requires users to take additional steps.

I agree “local” and “remote” is relative and thus ambiguous, but couldn’t come up with a better way of saying “on the MicroShift machine” vs “not on the MicroShift machine”.